pkg/eval/builtin_fn_io.go

```diff
@@ -673,6 +673,7 @@ func fromJSONInterface(v interface{}) (interface{}, error) {
 	case nil, bool, string:
 		return v, nil
 	case float64:
+		// TODO: Decide if we want to normalize ints here.
```

Comment:
JSON only has one number type and it corresponds to Go's float64; nothing can be done here to recover the "original" type.